src/renderer/components/Visualiser/store/actions.js

```diff
@@ -155,6 +155,7 @@ export default {
 
       graknTx.close();
 
+      window.visFacade = state.visFacade;
```

Comment:
Uh, let's make sure to remove this before merging

Reply:
this bit is actually required by some of the e2e tests. I can explain in person the current implementation and why this is needed and if we agree for this to be the right approach, then I will make sure the subject line only runs in `test` environment.